Gemfile.lock

```diff
@@ -314,6 +314,3 @@ DEPENDENCIES
   vcr
   webmock (~> 1.21)
   yajl-ruby (~> 1.2)
```

Comment:
Which version of bundler are you using locally? It seems kinda old given the `BUNDLED_WITH` stanza disappeared. You probably want to use `1.10.6`.